websites/D/Dark Wave/metadata.json

```
@@ -0,0 +1,21 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.13",
+  "apiVersion": 1,
+  "author": {
+    "id": "424317025712603140",
+    "name": "ForgeOff"
+  },
+  "service": "Dark Wave",
+  "description": {
+    "en": "a streaming service for looking films and series."
```

Use the official description from the service
```suggestion
    "en": "Stream your favorite movies and shows in high quality with zero ads. DarkWave offers a seamless, distraction-free entertainment experience."
```